runtime/v2/kata/options/kata.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package containerd.kata.v2;
+
+import weak "gogoproto/gogo.proto";
+
+option go_package = "github.com/containerd/containerd/runtime/v2/kata/options;options";
+
+message Options {
+	// specify the configure file used by kata
```

Comment:
`// ConfigPath specifies the filesystem location of the configuration file use by the kata runtime.

Reply:
updated.